src/App.tsx

```diff
@@ -6,7 +6,7 @@ import { NavigationBar } from "./Components/Navbar";
 import { HomePage } from "./Pages/Home";
 import { DonationPage } from "./Pages/Donation";
 import { Contact } from "./Pages/Contact";
-import { VisitUs } from "./Pages/VisitUs";
+import { VisitUs } from "./Pages/Visitus";
```

Comment:
Delete the VisitUs folder and create a PR and merge to the main branch. Then again create a PR with the VisitUs package. It should fix this issue